README.md

```diff
@@ -1,2 +1,32 @@
-# 2024-arrow-format-tutorial
-Repository for the Arrow Format Tutorial for PyCon DE 2024
+#  A deep dive into the Arrow Columnar format with pyarrow and nanoarrow
+
+Repository for the Arrow Columnar Format Tutorial at PyCon DE 2024.
+
+Apache Arrow has become a de-facto standard for efficient in-memory columnar data representation. You might have heard about Arrow or using Arrow, but do you understand the format and why it’s so useful? This tutorial will dive deep into the details of the Arrow columnar format, the different types and buffer layouts, and explore those details interactively using the pyarrow and nanoarrow libraries.
```

Comment:
```suggestion
Apache Arrow has become a de-facto standard for efficient in-memory columnar data representation. You might have heard about Arrow or using Arrow, but do you understand the format and why it's so useful? This tutorial will dive deep into the details of the Arrow columnar format, the different types and buffer layouts, and explore those details interactively using the pyarrow and nanoarrow libraries.
```
Not sure why it's using right single quotation mark instead of apostrophe

Reply:
Probably from copying this from the PyConDE website